include/spirv-tools/optimizer.hpp

```diff
@@ -201,15 +201,24 @@ Optimizer::PassToken CreateEliminateDeadConstantPass();
 // these is left for future improvements. 
 Optimizer::PassToken CreateBlockMergePass();
 
-// Creates an inline pass.
-// An inline pass exhaustively inlines all function calls in all functions
-// designated as an entry point. The intent is to enable, albeit through
-// brute force, analysis and optimization across function calls by subsequent
-// passes. As the inlining is exhaustive, there is no attempt to optimize for
-// size or runtime performance. Functions that are not designated as entry
-// points are not changed.
+// Creates an exhaustive inline pass.
+// An inline pass attempts to exhaustively inline all function calls in
+// all functions in an entry point call tree. The intent is to enable,
+// albeit through brute force, analysis and optimization across function
+// calls by subsequent optimization passes. As the inlining is exhaustive,
+// there is no attempt to optimize for size or runtime performance. Functions
+// that are not in the call tree of an entry point are not changed.
 Optimizer::PassToken CreateInlineExhaustivePass();
   
+// Creates an opaque inline pass.
+// An inline pass inlines all function calls in all functions in an entry
```

Comment:
Well, it's not that inefficient if I catch it.  :-)
We're probably in the right spot tradeoff-wise.